KMaina/Fast-Food-App
# #160298197 Page to View All Fast Food Items

#### What does this PR do?
Allow an admin to see all food items present
#### Description of Task to be completed?
The admin should be able to

- See all food items present

- Edit a food item by clicking on the edit button

- Delete a food item by clicking on a food item.

#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run `cd Fast-Food-App`

- run `git checkout -b ft-all-food-items-160298197`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-all-food-items-160298197`

- In the folder where you cloned the repo, you should open the UI folder

- Open the admin-food.html to see it in your browser for review

#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160298197
